README.md

```diff
@@ -26,7 +26,9 @@ Custom docker images need to use a Terra base image (see above) in order to work
 * Use the published container image location when creating notebook runtime
 * Dockerhub image example: [image name]:[tag]
 * GCR image example: us.gcr.io/repository/[image name]:[tag]
-    
+* Since 6/28/2021, we introduced a few changes that might impact building custom images
+    - Home directory of new images will be `/home/jupyter`. This means if your dockerfile is referencing `/home/jupyter-user` directory, you need to update it to $HOME (recommended) or `/home/jupyter`.
+    - Creating VMs with custom images will take much longer than `terra-jupyter-*` images because `docker pull` will take a few min. If the custom image ends up being too large, VM creation may time out. New base images are much larger in size than previous versions.
```

Comment:
can change `terra-jupyter-*` to `terra supported` if we decide to keep openVino image's name as is